src/java.base/share/classes/java/io/Reader.java

```diff
@@ -391,6 +396,70 @@ public int read(char[] cbuf) throws IOException {
      */
     public abstract int read(char[] cbuf, int off, int len) throws IOException;
 
+    /**
+     * Reads all remaining lines of text. A line is considered to be terminated
+     * by any one of a line feed ('\n'), a carriage return ('\r'), a carriage
+     * return followed immediately by a line feed, or by reaching the
+     * end-of-file (EOF).
```

Comment:
Reader is for reading characters so I think the starting line will say that is reads all remaining characters as lines of text.

In any case, I think you've enough here to help decide if a method should be added to Reader. I think the main observation is that it doesn't require Reader to be buffering.

Reply:
The description of the method should be the same in Reader and BufferedReader with respect to the behavior, line break handling, etc. It should be possible to use {@inheritDoc} to re-use it in BufferedReader.
Except for the description of the implementation, that can be in an {@impSpec} paragraph.
The method is consuming characters from the stream, assembling lines and returning list.